packages/ui-components/src/chat/chat.tsx

```diff
@@ -146,7 +146,8 @@ function Chat(props: ChatProps): JSX.Element {
             // Create a new message
             const newMessage = {
                 id: nanoid(),
-                message: reply,
+                // remove any /n and trailing whitespace
+                message: reply.replace(/\n/g, ' ').replace(/\s+$/, ''),
```

Comment:
Can we not just use trim here? Same in other places

Reply:
oops yes, fixed!